scripts/extension/__tests__/report.test.js

```diff
@@ -32,7 +38,8 @@ describe("muslOverride", () => {
   }
 
   function setHasMusl(value) {
-    mockHelpers.hasMusl.mockReturnValue(value)
+    const stderr = value ? "musl" : "something else"
```

Comment:
```suggestion
    const stderr = value ? "musl libc (arch)" : "something else"
```
Make so that it reflects the real world closer and it needs the substring check, and not `==`, for testing completeness.